homeassistant/components/media_player/plex.py

```diff
@@ -192,6 +200,20 @@ def update_devices():
             client.set_availability(client.machine_identifier
                                     in available_client_ids)
 
+            if not config.get(CONF_REMOVE_UNAVAILABLE_CLIENTS) \
+                    or client.available:
+                continue
+
+            if (dt.now() - client.marked_unavailable).total_seconds() >= \
+                    (config.get(CONF_CLIENT_REMOVE_INTERVAL)).total_seconds():
+                hass.helpers.event.async_call_later(0,
```

Comment:
- You can't call async methods inside sync context.
- Don't use call_later with 0. Use `hass.add_job` instead.

Reply:
ok

Reply:
Still looking at your previous comments about async etc. I'm not ignoring them, just trying to push the changes as they "work" while i figure all the other bits out.. 

I intend (not in this PR) to make `update_devices()` and `update_sessions()` async

Reply:
Yeah you can do the async stuff in a future PR